variables.tf

```diff
@@ -1,36 +1,15 @@
 # ECR variables
 
-variable "container_name" {
+variable "ecr_repo_url" {
   type        = string
-  description = "Container name"
+  description = "The URL of the github actions ECR repository"
+  default     = "037370603820.dkr.ecr.us-east-1.amazonaws.com/github-actions-runner"
```

Comment:
since this module is public, do we want this default to exist here, vs. in the runbook?

Reply:
I'm okay with leaving it since this could ultimately be in a public ECR registry/Github Package and could simplify the runbook slightly.